src/fragments/start/getting-started/reactnative/integrate.mdx

```diff
@@ -0,0 +1,411 @@
+  Text,
+  TextInput,
+  View,
+} from 'react-native';
+import { DataStore } from 'aws-amplify';
+import { Todo } from './models';
+
+const Header = () => (
+  <View style={styles.headerContainer}>
+    <Text style={styles.headerTitle}>My Todo List</Text>
+  </View>
+);
+
+const AddTodoModal = ({ modalVisible, setModalVisible }) => {
+  const [name, setName] = useState('');
+  const [description, setDescription] = useState('');
+
+  async function addTodo() {
+  }
+
+  function closeModal() {
+    setModalVisible(false);
+  }
+
+  return (
+    <Modal
+      animationType="fade"
+      onRequestClose={closeModal}
+      transparent
+      visible={modalVisible}
+    >
+      <View style={styles.modalContainer}>
+        <View style={styles.modalInnerContainer}>
+          <Pressable onPress={closeModal} style={styles.modalDismissButton}>
+            <Text style={styles.modalDismissText}>X</Text>
+          </Pressable>
+          <TextInput
+            onChangeText={setName}
+            placeholder="Name"
+            style={styles.modalInput}
+          />
+          <TextInput
+            onChangeText={setDescription}
+            placeholder="Description"
+            style={styles.modalInput}
+          />
+          <Pressable onPress={addTodo} style={styles.buttonContainer}>
+            <Text style={styles.buttonText}>Save Todo</Text>
+          </Pressable>
+        </View>
+      </View>
+    </Modal>
+  );
+};
+
+const TodoList = () => {
+  const [todos, setTodos] = useState([]);
+
+  useEffect(() => {
+
+  }, []);
+
+  async function deleteTodo(todo) {
+  }
+
+  async function setComplete(updateValue, todo) {
+  }
+
+  const TodoItem = ({ item }) => (
+    <Pressable
+      onLongPress={() => {
+        deleteTodo(item);
+      }}
+      onPress={() => {
+        setComplete(!item.isComplete, item);
+      }}
+      style={styles.todoContainer}
+    >
+      <Text>
+        <Text style={styles.todoHeading}>{item.name}</Text>
+        {`\n${item.description}`}
+      </Text>
+      <Text
+        style={[styles.checkbox, item.isComplete && styles.completedCheckbox]}
+      >
+        {item.isComplete ? '✓' : ''}
+      </Text>
+    </Pressable>
+  );
+
+  return (
+    <FlatList
+      data={todos}
+      keyExtractor={({ id }) => id}
+      renderItem={TodoItem}
+    />
+  );
+};
+
+const Home = () => {
+  const [modalVisible, setModalVisible] = useState(false);
+
+  return (
+    <>
+      <Header />
+      <TodoList />
+      <Pressable
+        onPress={() => {
+          setModalVisible(true);
+        }}
+        style={[styles.buttonContainer, styles.floatingButton]}
+      >
+        <Text style={styles.buttonText}>+ Add Todo</Text>
+      </Pressable>
+      <AddTodoModal
+        modalVisible={modalVisible}
+        setModalVisible={setModalVisible}
+      />
+    </>
+  );
+};
+
+const styles = StyleSheet.create({
+  headerContainer: {
+    backgroundColor: '#4696ec',
+    paddingTop: Platform.OS === 'ios' ? 44 : null,
```

Comment:
Nit: Should this be `undefined` (or even 0) instead of null?

Reply:
Good catch. I think it will behave the same but it is better to use `0`.